src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -45,6 +46,8 @@ protected ScriptHost(ScriptHostConfiguration scriptConfig)
             NodeFunctionInvoker.UnhandledException += OnUnhandledException;
         }
 
+        public event EventHandler IsPrimaryChanged;
+
         public static bool InStandbyMode
         {
```

Comment:
This was part of the hot standby change made here https://github.com/Azure/azure-webjobs-sdk-script/pull/534
It's a flag that indicates whether the host is in standby mode, which means it's pre-warmed but not setup with an actual customer Function App settings/environment.